<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/images/dynex</normaloff>:/images/dynex</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="WalletGui::OverviewFrame" name="m_overviewFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::SendFrame" name="m_sendFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::ReceiveFrame" name="m_receiveFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::TransactionsFrame" name="m_transactionsFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WalletGui::AddressBookFrame" name="m_addressBookFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_wallets">
     <property name="title">
      <string>Recent wallets</string>
     </property>
    </widget>
    <addaction name="m_openWalletAction"/>
    <addaction name="menuRecent_wallets"/>
    <addaction name="m_backupWalletAction"/>
    <addaction name="m_closeWalletAction"/>
    <addaction name="separator"/>    
    <addaction name="m_createWalletAction"/>
    <addaction name="m_createNondeterministicWalletAction"/>
    <addaction name="separator"/>
    <addaction name="m_exitAction"/>
   </widget>
   <widget class="QMenu" name="menuWallet">
    <property name="title">
     <string>Wallet</string>
    </property>
    <addaction name="m_encryptWalletAction"/>
    <addaction name="m_changePasswordAction"/>
    <addaction name="m_resetAction"/>
    <addaction name="separator"/>
    <addaction name="m_proofAction"/>
    <addaction name="separator"/>
    <addaction name="m_showMnemonicSeedAction"/>
    <addaction name="m_showPrivateKey"/>
    <addaction name="separator"/>
    <addaction name="m_restoreFromMnemonicSeedAction"/>
    <addaction name="m_importKeysAction"/>
    <addaction name="m_importKeyAction"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="m_connectionSettingsAction"/>
    <addaction name="separator"/>
    <addaction name="m_startOnLoginAction"/>
    <addaction name="m_minimizeToTrayAction"/>
    <addaction name="m_closeToTrayAction"/>
    <addaction name="m_globalAddressBookAction"/>
    <addaction name="separator"/>
    <addaction name="m_openLogFileAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="m_aboutDynexAction"/>
    <addaction name="m_aboutQtAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWallet"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_overviewAction"/>
   <addaction name="m_sendAction"/>
   <addaction name="m_receiveAction"/>
   <addaction name="m_transactionsAction"/>
   <addaction name="m_addressBookAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_overviewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/overview</normaloff>:/icons/overview</iconset>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </action>
  <action name="m_sendAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="m_receiveAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/receive</normaloff>:/icons/receive</iconset>
   </property>
   <property name="text">
    <string>Receive</string>
   </property>
  </action>
  <action name="m_transactionsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/transactions</normaloff>:/icons/transactions</iconset>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </action>
  <action name="m_exitAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_addressBookAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
   </property>
   <property name="text">
    <string>Address Book</string>
   </property>
  </action>
  <action name="m_createWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create wallet</string>
   </property>
  </action>
  <action name="m_openWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_encryptWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Encrypt wallet</string>
   </property>
  </action>
  <action name="m_changePasswordAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
   <property name="toolTip">
    <string>Change password</string>
   </property>
  </action>
  <action name="m_aboutDynexAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About wallet</string>
   </property>
  </action>
  <action name="m_aboutQtAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="m_backupWalletAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backup wallet</string>
   </property>
  </action>
  <action name="m_startOnLoginAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start on system login</string>
   </property>
  </action>
  <action name="m_minimizeToTrayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
  <action name="m_closeToTrayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close to tray</string>
   </property>
  </action>
  <action name="m_globalAddressBookAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use global address book</string>
   </property>
  </action>
  <action name="m_importKeyAction">
   <property name="text">
    <string>Restore from private key</string>
   </property>
  </action>
  <action name="m_resetAction">
   <property name="text">
    <string>Reset (rescan) wallet</string>
   </property>
  </action>
  <action name="m_proofAction">
   <property name="text">
    <string>Proof of balance</string>
   </property>
  </action>
  <action name="m_showPrivateKey">
   <property name="text">
    <string>Show private keys</string>
   </property>
  </action>
  <action name="m_closeWalletAction">
   <property name="text">
    <string>Close wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="m_recentWalletsAction">
   <property name="text">
    <string>Recent wallets</string>
   </property>
  </action>
  <action name="m_openLogFileAction">
   <property name="text">
    <string>Open log file</string>
   </property>
  </action>
  <action name="m_showMnemonicSeedAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show mnemonic seed</string>
   </property>
  </action>
  <action name="m_restoreFromMnemonicSeedAction">
   <property name="text">
    <string>Restore from mnemonic seed</string>
   </property>
  </action>
  <action name="m_createNondeterministicWalletAction">
   <property name="text">
    <string>Create nondeterministic wallet</string>
   </property>
  </action>
  <action name="m_importKeysAction">
   <property name="text">
    <string>Restore from keys</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="m_connectionSettingsAction">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::OverviewFrame</class>
   <extends>QFrame</extends>
   <header>gui/OverviewFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::SendFrame</class>
   <extends>QFrame</extends>
   <header>gui/SendFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::ReceiveFrame</class>
   <extends>QFrame</extends>
   <header>gui/ReceiveFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::TransactionsFrame</class>
   <extends>QFrame</extends>
   <header>gui/TransactionsFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WalletGui::AddressBookFrame</class>
   <extends>QFrame</extends>
   <header>gui/AddressBookFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_overviewAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_overviewFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_sendFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_receiveAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_receiveFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_transactionsAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_transactionsFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_addressBookFrame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_createWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_openWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_encryptWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>encryptWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_changePasswordAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>encryptWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_aboutQtAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backupWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>backupWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_aboutDynexAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_startOnLoginAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setStartOnLogin(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_minimizeToTrayAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMinimizeToTray(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_closeToTrayAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setCloseToTray(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_globalAddressBookAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setGlobalAddressBook(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_resetAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_proofAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>balanceProof()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_importKeyAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_showPrivateKey</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPrivateKeys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_openLogFileAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openLogFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_showMnemonicSeedAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMnemonicSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_restoreFromMnemonicSeedAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>restoreFromMnemonicSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_createNondeterministicWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNonDeterministicWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_importKeysAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importKeys()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_closeWalletAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeWallet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>DisplayCmdLineHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_connectionSettingsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openConnectionSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createWallet()</slot>
  <slot>openWallet()</slot>
  <slot>encryptWallet()</slot>
  <slot>aboutQt()</slot>
  <slot>backupWallet()</slot>
  <slot>about()</slot>
  <slot>setStartOnLogin(bool)</slot>
  <slot>setMinimizeToTray(bool)</slot>
  <slot>setCloseToTray(bool)</slot>
  <slot>setGlobalAddressBook(bool)</slot>
  <slot>importKey()</slot>
  <slot>resetWallet()</slot>
  <slot>showPrivateKeys()</slot>
  <slot>openLogFile()</slot>
  <slot>DisplayCmdLineHelp()</slot>
  <slot>openConnectionSettings()</slot>
  <slot>showMnemonicSeed()</slot>
  <slot>restoreFromMnemonicSeed()</slot>
  <slot>createNonDeterministicWallet()</slot>
  <slot>importKey()</slot>
  <slot>importKeys()</slot>
  <slot>closeWallet()</slot>
  <slot>balanceProof()</slot>
 </slots>
</ui>
